modules/persistent-storage-csi-ovirt-dynamic-provisioning.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-rhv.adoc
+
+[id="persistent-storage-csi-ovirt-dynamic-provisioning_{context}"]
+= Dynamically provisioning {rh-virtualization} CSI volumes
+
+{product-title} installs a StorageClass for each available oVirt share type.
```

Comment:
What does "oVirt share type" means ?
The storage class is created by the operator and its in the OCP cluster